src/platform/packages/shared/response-ops/rule_form/src/rule_definition/rule_investigation_guide_editor.tsx

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
@justinkambic I would move the file within `rule_details` folder

Reply:
Agree

Reply:
[2f9ce4e](https://github.com/elastic/kibana/pull/217106/commits/2f9ce4eccb81fc9bc6ccfb9937115794e57514c0)